src/ImageSharp/MetaData/Profiles/ICC/TagDataEntries/IccViewingConditionsTagDataEntry.cs

```diff
@@ -86,14 +86,11 @@ public override bool Equals(object obj)
         /// <inheritdoc/>
         public override int GetHashCode()
         {
-            unchecked
-            {
-                int hashCode = base.GetHashCode();
-                hashCode = (hashCode * 397) ^ this.IlluminantXyz.GetHashCode();
-                hashCode = (hashCode * 397) ^ this.SurroundXyz.GetHashCode();
-                hashCode = (hashCode * 397) ^ (int)this.Illuminant;
-                return hashCode;
-            }
+            return HashHelpers.Combine(
+                base.GetHashCode(),
+                this.IlluminantXyz.GetHashCode(),
+                this.SurroundXyz.GetHashCode(),
+                (int)this.Illuminant);
```

Comment:
`this.Illuminant.GetHashCode()`?

This is happening in other places also, now starting to wonder if you should not use GetHashCode for an int?

Reply:
Going to update these all to using GetHashCode() for consistency.